metron-analytics/metron-profiler-common/src/main/java/org/apache/metron/profiler/hbase/SaltyRowKeyBuilder.java

```diff
@@ -241,10 +241,18 @@ public SaltyRowKeyBuilder(int saltDivisor, long duration, TimeUnit units) {
     }
   }
 
+  public long getPeriodDurationMillis() {
+    return periodDurationMillis;
+  }
+
   public void withPeriodDuration(long duration, TimeUnit units) {
     periodDurationMillis = units.toMillis(duration);
   }
 
+  public int getSaltDivisor() {
```

Comment:
This is what generates the row keys specifically for profiles stored in HBase.  It does not relate to Enrichments at all.

Reply:
I understand it's not related to enrichments, however there are some common/standard practices with HBase, e.g. salting, that are common to both. This is legacy code, so I recant my earlier request to have this done before the upgrade is finished. My only ask would be to potentially create a Jira out of band from the FB that calls out something like "Reconcile/refactor common hbase idioms in code base"

Reply:
I don't have enough context around this to create a JIRA with enough detail to be useful.  I can't even find where salting occurs in Enrichment so as to document a concrete example.  Would you mind creating the JIRA since you have a better idea of what we might need?

Reply:
This seems like an exploratory Jira, first. The salting would have come along with some of the LookupKV stuff, I'd imagine. I haven't been in there a while aside from reviewing the HBase upgrade PRs - did you not come across any of that while doing the upgrade? I can't imagine it could have been avoided.

Reply:
Nothing about salting needed changed, so I can't remember running across that during the upgrade work.

Reply:
I don't have much to add here. We can address this at a later time I suppose.